languages/swift/exercises/concept/string-components/.docs/hints.md

```diff
@@ -24,8 +24,19 @@
 
 - String interpolation or concatenation can be used to glue two strings together.
 
+## 7. Get the i<sup>th</sup> letter of a sentence
+
+- Swift has a [method][string-index-offset] for computing an index based that is offset from another index.
+- The `startIndex` property of a String returns an index to the first character of the string.
```

Comment:
Maybe rephrase slightly and link to an external resource to not directly give away the property to use?